ui/src/views/Settings.vue

```diff
@@ -22,16 +22,283 @@
+                    "
+                    v-model="isModMamStatus"
+                    :disabled="
+                      loading.getConfiguration || loading.configureModule
+                    "
+                    class="mg-bottom"
+                  >
+                    <template slot="text-left">{{
+                      $t("settings.disabled")
+                    }}</template>
+                    <template slot="text-right">{{
+                      $t("settings.enabled")
+                    }}</template>
+                  </cv-toggle>
+                  <template v-if="isModMamStatus">
+                    <cv-toggle
+                      value="purge_mnesia_database"
+                      :label="$t('settings.Enable_purge_mnesia_database')"
+                      v-model="isPurgeMnesiaDatabase"
+                      :disabled="
+                        loading.getConfiguration || loading.configureModule
+                      "
+                      class="mg-bottom mg-left"
+                    >
+                      <template slot="text-left">{{
+                        $t("settings.disabled")
+                      }}</template>
+                      <template slot="text-right">{{
+                        $t("settings.enabled")
+                      }}</template>
+                    </cv-toggle>
+                    <template v-if="isPurgeMnesiaDatabase">
+                      <NsSlider
+                        class="mg-left"
+                        v-model="purge_mnesia_interval"
+                        :label="$t('settings.purge_mnesia_database_interval')"
+                        min="1"
+                        max="365"
+                        step="1"
+                        stepMultiplier="1"
+                        minLabel=""
+                        maxLabel=""
+                        :invalidMessage="error.purge_mnesia_interval"
+                        :disabled="
+                          loading.getConfiguration || loading.configureModule
+                        "
+                      />
+                    </template>
+                  </template>
+                  <cv-toggle
+                    value="http_upload"
+                    :label="$t('settings.Enable_file_upload_mod_http_upload')"
+                    v-model="isHttpUploadEnabled"
+                    :disabled="
+                      loading.getConfiguration || loading.configureModule
+                    "
+                    class="mg-bottom"
+                  >
+                    <template slot="text-left">{{
+                      $t("settings.disabled")
+                    }}</template>
+                    <template slot="text-right">{{
+                      $t("settings.enabled")
+                    }}</template>
+                  </cv-toggle>
+                  <template v-if="isHttpUploadEnabled">
+                    <cv-toggle
```

Comment:
Same as above, I would invert this logic: instead of using `CvToggle` I would use a `NsSlider` with label `Keep old files for (days)`. See `https://github.com/NethServer/ns8-core/blob/main/core/ui/src/stories/NsSlider.stories.js` or try it live on storybook

Reply:
done